src/engine/fileOpsIns.cpp

```diff
@@ -631,6 +632,77 @@ void DivEngine::loadSBI(SafeReader& reader, std::vector<DivInstrument*>& ret, St
   ret.push_back(ins);
 }
 
+void DivEngine::loadFF(SafeReader& reader, std::vector<DivInstrument*>& ret, String& stripPath) {
+  DivInstrument* insList[256];
+  memset(insList,0,256*sizeof(void*));
+  int readCount = 0;
+  size_t insCount = reader.size();
+  insCount = (insCount >> 5) + (((insCount % 0x20) > 0) ? 1 : 0);
+  if (insCount > 256) insCount = 256;
+  uint8_t buf;
+  try {
+    reader.seek(0, SEEK_SET);
+    for (unsigned int i = 0; i < insCount; ++i) {
+      insList[i] = new DivInstrument;
+      DivInstrument* ins = insList[i];
+
+      ins->type = DIV_INS_FM;
+      DivInstrumentFM::Operator op;
+
+      for (unsigned int j = 0; j < 4; j++) {
+        buf = reader.readC();
+        ins->fm.op[j].mult = buf & 0xf;
+        ins->fm.op[j].dt = [](uint8_t dt){if (dt >= 4) { return 7 - dt; } else { return dt + 3; }}((buf >> 4) & 0x7);
```

Comment:
Ternary no good here?

`ins->fm.op[j].dt = [](uint8_t dt){return (dt >= 4) ? (7 - dt) : (dt + 3); }}((buf >> 4) & 0x7);`

Reply:
Fixed, used the suggestion from https://github.com/OPNA2608/furnace/commit/d792c80064d10578d237d770d4f9c792faddb32c#r70909499.